basket/src/components/Hello.tsx

```diff
@@ -0,0 +1,9 @@
+export const Hello = (props: { name: string }) => {
+  return <h1>Hello, {props.name}!!</h1>
+}
```

Comment:
1. 디스트럭처링을 사용해서 아래와 같이 작성하기!
2. 컴포넌트의 props 타입은 별도로 타입 선언하기!

```suggestion
type HelloProps = {
  name: string;
}

export const Hello = ({ name }: HelloProps) => {
  return <h1>Hello, {name}!!</h1>
}
```